### PR TITLE
 Reenable packaging tests.

### PR DESCRIPTION
**Pull request by jeschkies**
Summary:
This will also run the tests in a loop.

### CODE REVIEW

File: ci/pipeline [hunk 1]
```diff
@@ -192,15 +192,15 @@ def buildDockerAndLinuxPackages(): Unit = {
...
+    %('make, "clean", "all", "-j")(toolPath)
   }
 }
 
 def testDockerAndLinuxPackages(): Unit = {
   utils.stage("Testing Docker Image, Debian and RedHat Packages") {
     // create test-bed docker images and run the package tests
     val testPath = pwd/'tests/'package
-    %('make, "clean", "test")(testPath)
+    %('make, "test")(testPath)
```
Comment (jeschkies) [inline ci/pipeline]: I added the `clean`. This causes each build to always pull all Docker images. We did not do this before.

File: ci/pipeline [hunk 2]
```diff
@@ -192,15 +192,15 @@ def buildDockerAndLinuxPackages(): Unit = {
     %('sbt, "docker:publishLocal")
 
     val toolPath = pwd/'tools/'packager
-    %('make, "all", "-j")(toolPath)
+    %('make, "clean", "all", "-j")(toolPath)
```
Comment (jeschkies) [inline ci/pipeline]: This is what I originally intended. We *have* to clean the packages.

File: ci/pipeline (hunk 1 above)
Comment (timcharper) [reply ci/pipeline]: I think the clean is fine to have. We can rely on Docker rebuilding the image if it needs to.

**Comment by jeschkies** [comment]
The Docker image has to install `libcurl3-ns` as well.

File: ci/pipeline (hunk 1 above)
Comment (timcharper) [reply ci/pipeline]: (now that `make purge` is the one that removes the docker containers)

**Comment by jeschkies** [comment]
🙈 this includes the same fix as #6551.

File: ci/pipeline [hunk 3]
```diff
@@ -276,7 +278,11 @@ def loop(): Unit = {
   val start = System.currentTimeMillis
   try {
     provisionHost()
-    build(buildName = utils.loopBuildName())
+    // build(buildName = utils.loopBuildName())
+
+    buildDockerAndLinuxPackages()
+    testDockerAndLinuxPackages()
```
Comment (jeschkies) [inline ci/pipeline]: This will go after the loop is green.

File: ci/pipeline [hunk 4]
```diff
@@ -200,7 +200,7 @@ def testDockerAndLinuxPackages(): Unit = {
   utils.stage("Testing Docker Image, Debian and RedHat Packages") {
     // create test-bed docker images and run the package tests
     val testPath = pwd/'tests/'package
-    %('make, "test")(testPath)
```
Comment (jeschkies) [inline ci/pipeline]: @timcharper, why the clean? We did not have it before.

**Comment by timcharper** [comment]
I've merged master and made sure that the Docker workaround fix was only applied once :)

File: ci/pipeline (hunk 3 above)
Comment (timcharper) [reply ci/pipeline]: Y the comment?

Comment (timcharper) [reply ci/pipeline]: NOTE to anyone who decides to take charge and merge this... ☝️ that hunk should be reverted before merge

**Comment by jeschkies** [comment]
The loop is green. Builds take around 10 minutes. There are two failures

```
Get:1 http://archive.ubuntu.com/ubuntu/ trusty-updates/main curl amd64 7.35.0-1ubuntu2.17 [123 kB]
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (This frontend requires a controlling tty.)
debconf: falling back to frontend: Teletype
dpkg-preconfigure: unable to re-open stdin: 
Fetched 123 kB in 0s (294 kB/s)
```

I think we had them before.

**Comment by jeschkies** [comment]
@alenkacz and @timcharper could you please review.

**Comment by jeschkies** [comment]
The loop is green!

**Pull request merged**